delete wasm feature and dependencies, as those are not packaged and not needed

diff --git a/Cargo.toml b/Cargo.toml
index 9f2948c..d3514ab 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -25,33 +25,5 @@ keywords = [
 license = "MIT OR Apache-2.0"
 repository = "https://github.com/1Password/sys-locale"
 
-[features]
-js = [
-    "js-sys",
-    "wasm-bindgen",
-    "web-sys",
-]
-
-[target."cfg(all(target_family = \"wasm\", not(unix)))".dependencies.js-sys]
-version = "0.3"
-optional = true
-
-[target."cfg(all(target_family = \"wasm\", not(unix)))".dependencies.wasm-bindgen]
-version = "0.2"
-optional = true
-
-[target."cfg(all(target_family = \"wasm\", not(unix)))".dependencies.web-sys]
-version = "0.3"
-features = [
-    "Window",
-    "WorkerGlobalScope",
-    "Navigator",
-    "WorkerNavigator",
-]
-optional = true
-
-[target."cfg(all(target_family = \"wasm\", not(unix)))".dev-dependencies.wasm-bindgen-test]
-version = "0.3"
-
 [target."cfg(target_os = \"android\")".dependencies.libc]
 version = "0.2"
